packages/office-ui-fabric-react/src/components/Button/CompoundButton/CompoundButton.styles.ts

```diff
@@ -41,17 +47,26 @@ export const getStyles = memoizeFunction((
 
     label: {
       margin: '0 0 5px',
-      lineHeight: '100%'
+      lineHeight: '100%',
+      fontWeight: FontWeights.semibold
     },
-
     description: [
       theme.fonts.small,
       {
-        color: theme.palette.neutralSecondary,
         lineHeight: '100%'
       }
     ],
 
+  };
+
+  let standardCompoundTheme: IButtonStyles = {
+
+    ...standardStyles(theme),
```

Comment:
I guess that's the point. standardStyles are the common styles between all of the standard type buttons. Compound buttons are special in that they have descriptions. I wouldn't want to add the additional styling to standardStyles as that'd increase the weight of all of the buttons that used it.